translators/sqlite_meta.go

```diff
@@ -111,7 +111,9 @@ func (p *sqliteSchema) buildTableData(table *fizz.Table, db *sql.DB) error {
 }
 
 func (p *sqliteSchema) buildTableIndexes(t *fizz.Table, db *sql.DB) error {
-	prag := fmt.Sprintf(`SELECT "seq", "name", "unique", "origin", "partial" FROM pragma_index_list('%s')`, t.Name)
+	// This ignores all internal SQLite keys which are prefixed with `sqlite_` as explained here:
+	// https://www.sqlite.org/fileformat2.html#intschema
+	prag := fmt.Sprintf(`SELECT "seq", "name", "unique", "origin", "partial" FROM pragma_index_list('%s') WHERE "name" NOT LIKE 'sqlite%%'`, t.Name)
```

Comment:
Just a little detail: `SQLite prohibits applications from creating objects whose names begin with "sqlite_". ` So here, I guess it should be `WHERE "name" NOT LIKE 'sqlite_%%'` (`sqlitey` seems to be a valid prefix, if I read the docs correctly).